_includes/parse-server/experimental.md

```diff
@@ -0,0 +1,390 @@
+# Experimental
+
+Experimental features may not be 'battle tested' and could have issues. They are likely to become full features at some point, but could be removed from the system if they are problematic.
```

Comment:
Much better that my earlier suggestions! I have a couple little suggestions on top of your suggestion...

Remove `experimental` -> using the word in the definition
Remove `without phased deprecation` -> I don't think we've ever done phased deprecation so could give people the wrong idea.

> The following features are still under active development and subject to an increased probability of bugs, breaking changes, non-backward compatible changes or removal in any future version. It is not recommended to use these features in production or other critical environments.
>
> However, we strongly suggest and appreciate if you try out these features in development environments and provide feedback, so that they can mature faster and become production fit.

Reply:
Sadly, you are right, although I am injecting phased deprecation into conversations wherever I can recently.